tests/test_interface_private_vlan.rb

```diff
@@ -321,6 +321,14 @@ def test_switchport_pvlan_trunk_allowed_vlan
 
     i.switchport_pvlan_trunk_allowed_vlan = default
     assert_equal(default, i.switchport_pvlan_trunk_allowed_vlan)
+
+    vlans = '500-528,530,532,534,587,590-593'
+    all_vlans = '500-528,530,532,534,587,590-593,597-598'
+    config("interface #{i.name}",
+           "switchport private-vlan trunk allowed vlan #{vlans}",
```

Comment:
As far as I can tell, `switchport private_vlan trunk allowed vlan` does not have the multi-line behavior that `switchport trunk allowed vlan` does, but I added in the logic to handle it if the behavior is added in later on.